fre/pp/configure_script_yaml.py

```diff
@@ -1,71 +1,82 @@
 """
-    Script creates rose-apps and rose-suite
-    files for the workflow from the pp yaml.
+Script creates rose-apps and rose-suite
+files for the workflow from the pp yaml.
 """
 
-## TO-DO:
-# - condition where there are multiple pp yamls
-# - validation here or in combined-yamls tools
-
 import os
 import json
+import logging
 
 from pathlib import Path
+from typing import Tuple
 from jsonschema import validate, SchemaError, ValidationError
-import yaml
 import metomi.rose.config
 
 import fre.yamltools.combine_yamls_script as cy
 
-import logging
 fre_logger = logging.getLogger(__name__)
 
-####################
-def yaml_load(yamlfile):
-    """
-    Load the given yaml.
-    """
-    # Load the main yaml
-    with open(yamlfile,'r') as f:
-        y=yaml.safe_load(f)
-
-    return y
-
 ######VALIDATE#####
-def validate_yaml(yamlfile):
+def validate_yaml(yamlfile: dict):
     """
-     Using the schema.json file, the yaml format is validated.
+    Using the schema.json file in gfdl_msd_schemas,
+    the yaml format is validated.
```

Comment:
Validate the format of the yaml file based on the schema.json file in gfdl_msd_schemas..?